src/core/model_config.proto

```diff
@@ -146,6 +203,13 @@ message ModelInstanceGroup
   //@@     of instances created. Default is 1.
   int32 count = 2;
 
+  //@@  .. cpp:var:: ModelRateLimiter rate_limiter
+  //@@
+  //@@     The rate limiter specific settings to be associated with this
+  //@@     instance group.
```

Comment:
"Optional, if not specified no rate limiting will be applied to this instance group."